src/Nethermind/Nethermind.Network/ProtocolsManager.cs

```diff
@@ -250,7 +250,8 @@ private void InitSyncPeerProtocol(ISession session, SyncPeerProtocolHandlerBase
                     if (_syncPeers.TryAdd(session.SessionId, handler))
                     {
                         _syncPool.AddPeer(handler);
-                        _txPool.AddPeer(handler);
+                        if (handler.ProtocolCode != Protocol.Les)
```

Comment:
maybe add a property bool IncludeInTxPool on the handler instead of having the logic here